src/main/kotlin/nl/info/zac/zaak/ZaakService.kt

```diff
@@ -358,4 +363,48 @@ class ZaakService @Inject constructor(
         }
         zrcClientService.createRol(role, explanation)
     }
+
+    fun processSpecialBrondatumProcedure(zaak: Zaak, resultaatTypeUUID: UUID, brondatumArchiefprocedure: BrondatumArchiefprocedure) {
```

Comment:
```suggestion
    fun processBrondatumArchiefprocedure(zaak: Zaak, resultaatTypeUUID: UUID, brondatumArchiefprocedure: BrondatumArchiefprocedure) {
```
?

I find the word 'special' a bit confusing? This sounds to me like there is also a 'normal' way to process these brondatumarchiefprocedures?

Reply:
good point, will change